.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -109,3 +108,11 @@ body:
         Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
     validations:
       required: false
+
+  - type: checkboxes
+    attributes:
+      label: Latest Docker version
+      description: You should try to use latest Docker version
+      options:
+        - label: I've updated my Docker version to the latest available and the issue still persists
```

Comment:
We've seen several people, who were using old Docker versions and they were getting some errors. I think this should make them update the Docker version before creating an issue.

Reply:
See: https://github.com/jupyter/docker-stacks/issues/1856